src/modules/message/dto/sendMesasageFile.dto.ts

```diff
@@ -0,0 +1,18 @@
+import { IsString, IsNotEmpty } from 'class-validator';
+
+export class SendMessageFileDto {
+  @IsNotEmpty()
+  @IsString()
+  to: string;
+
+  @IsNotEmpty()
+  @IsString()
+  path: string;
+
+  @IsNotEmpty()
+  @IsString()
+  filename: string;
+
+  @IsString()
+  subtitle: string;
```

Comment:
esse aqui é obrigatório ou opcional?

Reply:
só é obrigatório no método de enviar o vídeo como gif

Reply:
saquei

Reply:
esse subtitle eu tô forçando ser o nome do arquivo pq não vem na requisição

Reply:
![Screenshot_20210605-145845_WhatsApp.jpg](https://user-images.githubusercontent.com/54558516/120901082-fcea7e00-c60e-11eb-8c33-9cb1f99d3c41.jpg)

Reply:
saquei, vamos ter q arrumar um jeito pra conseguir enviar esse subtitle vazio. Não da pra enviar assim como tu me mandou no print.

Talvez adicionar um campo subtitle opcional na request POST /messages

Reply:
tenta enviar '' no campo

Reply:
blz

Reply:
![Screenshot_20210605-174648_WhatsApp.jpg](https://user-images.githubusercontent.com/54558516/120905078-13e89a80-c626-11eb-84ba-d844adc31b97.jpg)

Reply:
ficou desse jeito ai

Reply:
boa melhorou!
depois vamos ter que receber esse subtitle pela request, porque o usuário pode enviar um video com uma descrição pelo mchat

Reply:
Se tu quiser fazer agora tbm, pode colocar. ele seria um parametro opcional

Reply:
```suggestion
  @IsOptional()
  @IsString()
  subtitle?: string = '';
```

Reply:
deixa ele assim ae

Reply:
Ai ja fica default uma string vazia